### ISSUE TITLE
White space to left of captured image 

### ISSUE THREAD

**Issue by jimiswells** [niklasvh/html2canvas#2510]
Please make sure you are testing with the latest [release of html2canvas](https://github.com/niklasvh/html2canvas/releases). 
Old versions are not supported and issues reported for them will be closed.

# Please follow the general troubleshooting steps first:

- [ ] You are using the latest [version](https://github.com/niklasvh/html2canvas/releases)
- [ ] You are testing using the non-minified version of html2canvas and checked any potential issues reported in the console

### Bug reports:

On some occasions I'm getting a thick white line to the left of the image displaying in my div which is captured from another div offscreen.. capture code below - 

 html2canvas(document.querySelector("#capture"), {
      scrollX: -window.scrollX,
      scrollY: -window.scrollY,
      windowWidth: document.documentElement.offsetWidth,
      windowHeight: document.documentElement.offsetHeight,
    }).then((canvas) => {
      window.scrollTo(0, 0);
      var data = canvas.toDataURL("image/png");
      var img = $(`<img src="${data}" id="dynamic">`);

      $("#displayCanvas").addClass("imgRendered");
      $("#displayCanvas").addClass("bg-grey");
      $("#displayCanvas").append(img);
    });

### Specifications:

 * html2canvas version tested with:
 * Browser & version:
 * Operating system:

**Comment by pagarevijayy** [comment]
same but on the bottom.

**Comment by 553082694** [comment]
window.scrollTo(0, 0);

before html2canvas()

**Comment by valtism** [comment]
Yep, seems to be because of the scroll. This is annoying because when I have a mouse plugged in (macOS) the presence of the fixed scrollbar on the right means that all of the images come out with a bar on their left. 

I was able to get around this by setting `document.documentElement.style.overflow = "hidden"` before the call, and `document.documentElement.style.overflow = ""` after.
